Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM rocker/verse:3.5.3@sha256:7cf4253239c338fbef924ddf46b7632cc52d1da7616e3d80b11bca0093071cb2
+FROM continuumio/miniconda3:4.8.2 as miniconda3
```

Comment:
What's the reason for using conda 4.8.2 instead of 4.8.3?

Reply:
4.8.3 isn't available yet which is kind of annoying otherwise I would have just used it :(